docs/source/classroom/class_guide.md

```diff
@@ -0,0 +1,172 @@
+(class-guide)=
+# Courses on HPC at NU
+
+We support classroom education at Northeastern University by providing access to computing resources (CPU and GPU) and storage resources for instructors and their students.
+
+We’ve supported courses from many disciplines, including biology, chemistry, civil engineering, machine learning, computer science, mathematics, and physics.
+
+To gain access to HPC resources instructors need to submit a [classroom access form.](https://service.northeastern.edu/tech?id=sc_cat_item&sys_id=0ae24596db535fc075892f17d496199c)
+
+Please submit these requests prior to the beginning of each semester (preferred), or at least **one week** prior to the start of when you plan on using the HPC cluster for your class.
```

Comment:
The one week line should be doable but definitely is a little worrisome if the environment or application is complicated.

Reply:
fixed this link